cli/run.go

```diff
@@ -0,0 +1,70 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
+	"os"
+)
+
+// RunOptions describe configuration options that can
+// be set using the run subcommand.
+type RunOptions struct {
+	*RootOptions
+	Config string
+}
+
+var clientRunExamples = templates.Examples(
+	`
+	# Push artifacts
+	client run ./config.yaml
+	`,
+)
+
+// NewRunCmd creates a new cobra.Command for the run subcommand.
+func NewRunCmd(rootOpts *RootOptions) *cobra.Command {
+	o := RunOptions{RootOptions: rootOpts}
+
+	cmd := &cobra.Command{
+		Hidden:        true, // TODO delete after implementation
```

Comment:
Instead of statically defining the hidden flag, what about reading from an environment variable (say `UOR_DEV_MODE` for example). If true, show it

Reply:
I like this idea too. It would save us from the eventuality where someone is developing or testing the run command and forgets to set `Hidden` back to true,